steps/github-step-repositories-describe/step.yaml

```diff
@@ -0,0 +1,60 @@
+# The schema version. Required. Must be exactly the string "integration/v1".
+apiVersion: integration/v1
+
+# The schema kind. Required. Must be one of "Query", "Step", or "Trigger"
+# corresponding to its directory location.
+kind: Step
+
+# The name of the action. Required. Must be exactly the name of the directory
+# containing the action.
+name: github-step-repositories-describe
+
+# The version of the action. Required. Must be an integer. If specified in the
+# directory name, must be exactly the version in the directory name.
+version: 1
+
+# High-level phrase describing what this action does. Required.
+summary: Describe GitHub repositories belonging to an org
+
+# Single-paragraph explanation of what this action does in more detail.
+# Optional. Markdown.
+description: |
+  Describe GitHub repositories belonging to an org that match given parameters.
+# URL or path relative to this file to an icon or icons representing this
+# action. Optional. Defaults to the integration icon.
+icon:
+
+# The mechanism to use to construct this step. Required. Must be an action
+# builder. See the Builders section below.
+build:
+  # The schema version for builders. Required. For now, must be the exact
+  # string "build/v1". We may consider supporting custom third-party builders
+  # in the future.
+  apiVersion: build/v1
+
+  # The builder to use. Required.
+  kind: Docker
+
+publish:
+  repository: relaysh/relay-github
```

Comment:
Unlike everything else that automatically infers the integration and step name together, this should be the full name of the Docker image for customers to use. So in this case, you'd want `relaysh/github-step-repositories-describe`.

Reply:
I thought this was the repository, not the image

Reply:
So this is technically a repository in Docker's jargon. The nuance is something like:

`index.docker.io` is a registry. `[index.docker.io/]foo/whatever` is a repository in that registry that "holds" images. Each tag in `foo/whatever`, e.g. `foo/whatever:latest`, `foo/whatever:v2` points to a content-addressable storage location that contains the actual image data. An image is basically a bunch of filesystem layers stacked on top of each other with some metadata. The concept of a user (e.g. `relaysh` in the repository `relaysh/github-step-whatever`) is Docker Hub-specific, and lots of Docker registries don't work that way (for example GCR/GAR use GCP project names instead).